lib/index.js

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const env = require('./env');
+
+const api = {};
+module.exports = api;
+
+let base64url;
+if(env.nodejs) {
+  // FIXME: the `encode` API for npm base64url allows string or buffer, should
+  // steps be taken homogenize the APIs in different libs, or consider it a
+  // feature?
```

Comment:
We should at least throw if `input` is not a Uint8Array/Buffer in the js-only version if we don't want to do something to support string inputs.

Reply:
done.